.gitignore

```diff
@@ -0,0 +1,101 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Please, remove this file. This kind of files are part of developer environment, if we have something project specific we add it here.